src/Check/ExtensionLoaded.php

```diff
@@ -21,33 +17,25 @@
  */
 class ExtensionLoaded extends AbstractCheck implements CheckInterface
 {
-    /** @var array|Traversable */
+    /** @var non-empty-list<string> */
     protected $extensions;
 
     /** @var bool */
     protected $autoload = true;
 
     /**
-     * @param  string|array|Traversable  $extensionName PHP extension name or an array of names
+     * @param  string|non-empty-list<string>  $extensionName PHP extension name or an array of names
      * @throws InvalidArgumentException
      */
-    public function __construct($extensionName)
+    public function __construct(string|array $extensionName)
```

Comment:
```suggestion
    public function __construct(string|iterable $extensionName)
```

Traversable was allowed previously.